draft-ietf-cats-framework.md

```diff
@@ -560,6 +560,16 @@ More importantly, the means for identifying a flow for the purpose of ensuring i
 
 This document does not define any mechanism for defining or enforcing service contact instance affinity.
 
+## Policy Delivery
+
+As shown in Figure 1, the control plane and management plane need to communicate some policies to the data plane. The control plane and management plane can be implemented in a control center entity.
+
+If the C-PS is logically centralized, it needs to communicate some policies to the CATS-Forwarders. In this case, the network entity containing the C-PS, such as a control center, should be able to perform management and/or control functions, including computing information awareness function, service scheduling function, etc. It collects computing and network information, and schedules service based on computing and network information. The CATS-Forwarders performs data plane functions or routing functions.
+
+If the C-PS is collocated with an Ingress CATS-Forwarder, the functions that the Ingress CATS-Forwarder supports would include computing information awareness function, service scheduling function and data forwarding function. In this case, Ingress CATS-Forwarders make forwarding decisions for a given service request that has been received from a client according to the capabilities and status information of both service instances and network.
```

Comment:
This text seems to redefine the function of an CATS-Forwarder.

Reply:
We do not want to redefined the function of the Ingress CATS-Forwarder. We want to state that the Ingress CATS-Forwarder can include the function of the C-PS, which makes a different situation for policy provision.